Lib/http/cookiejar.py

```diff
@@ -607,9 +615,10 @@ def user_domain_match(A, B):
     B = B.lower()
     if not (liberal_is_HDN(A) and liberal_is_HDN(B)):
         if A == B:
-            # equal IP addresses
+            # equal IPV4 addresses
```

Comment:
Please, leave this comment as it is currently. It is correct, we can have IPv4 *or* IPv6 addresses.